app/src/main/res/values/strings.xml

```diff
@@ -174,6 +174,12 @@
     <string name="hide_soft_when_physical_off_summary">Show soft keyboard when physical keyboard is
         used
     </string>
+    <string name="enable_alt_space_language_shortcut">Alt + Space switches languages</string>
```

Comment:
I believe we should use uppercase _SPACE_, and not _Space_. What do you think?

Reply:
I was trying to be consistent with the style of some of the other config options -- e.g. "Space switches keyboards": "Switch from symbols/numbers to alphabet when pressing space".
I've found a mix of "Space" and "space", but I don't see any "SPACE".

Reply:
Then capitalizrd `Space`. It's a name of  key.